docs/source/contributing/developer_guide_implementing_distribution.md

```diff
@@ -202,6 +213,10 @@ blah = pm.Blah.dist([0, 0], [1, 2])
 blah.eval()
 # array([0.62778803, 1.95165513])
 
+# Test the get_moment
+get_moment(blah).eval()
+# array([0.5, 1. ])
```

Comment:
These would be `array([0., 0.])` with the pseudo code above no?

Reply:
This is what I'm getting from `get_moment()` for `blah = pm.Uniform.dist([0, 0], [1, 2])` 

I did some tests and I think this is the right output. Considering that the first argument is location and second is scale (or lower and upper), then first interval is `[0, 1]` and second interval is `[0, 2]`. With the formula `mean = 0.5 * [a, b]`, then we have `[0.5, 1.0]`.
I compared with scipy implementation and got the same result.

```
import pymc as pm
from pymc.distributions.distribution import get_moment
from scipy.stats import uniform 

# PyMC implementation
blah = pm.Uniform.dist([0, 0], [1, 2])
get_moment(blah).eval()
>> array([0.5, 1. ])

# spicy implementation
uniform.mean([0, 0], [1, 2])
>> array([0.5, 1. ])
```

I see that there is an error in `pre-commit / pre-commit (pull_request) `. Is this something I did wrong? Let me know how I can fix it.

Thank you,

Reply:
Thanks for the clarification! Yes, I agree it makes sense to keep it uniform by changing this example from `Uniform` to `Normal`. I will commit the changes below.